### DISCUSSION TITLE
Reversed Scrolling from "Linear Mouse" app causes sliders to be reversed

### DISCUSSION THREAD

**Discussion by ismaiiil** [MonitorControl/MonitorControl#1491] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I use the reverse mouse scroll functionality because of natural scrolling with the touchpad and the mouse conflicts when enabling through "system preferences" which is why i had to download the app "Linear Mouse" to fix that (among other issues such as acceleration)

I'm just asking if the app could potentially handle scrolling to change volume the same way as every other app does and not be reversed? (when it has already been reversed)

Because right now, when i scroll up, the volume goes down, and scroll down the volume goes up.

**Comment by waydabber** [comment]
Unfortunately MonitorControl's feature set is final and no more updates are planned. However the source code is available, I recommend you modifying this part of the code and build a custom version. You can also try [BetterDisplay](https://betterdisplay.pro) as it handles swipe (left/right scroll) gesture as well which should not be affected by reverse scroll (you need to enable scroll/swipe in Settings/Menu).

**Reply by ismaiiil** [reply]
Okay, Thanks for the info ^^ , ill give it a try :D
